src/model/selection.js

```diff
@@ -625,6 +625,20 @@ export default class Selection {
 	}
 
 	/**
+	 * Checks whether the entire content in specified element is selected.
+	 *
+	 * @param {module:engine/model/element~Element} element
+	 * @returns {Boolean}
+	 */
+	isEntireContentSelected( element ) {
```

Comment:
This element should default to the root of the selection's root (`this.anchor.root` will be fine).